lib/components/form/error-message.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
-import { FormattedMessage } from 'react-intl'
+import { FormattedMessage, useIntl } from 'react-intl'
 import { connect } from 'react-redux'
 
 import TripTools from '../narrative/trip-tools'
 import Icon from '../util/icon'
 import { getActiveError, getErrorMessage } from '../../util/state'
 
-const ErrorMessage = ({ message }) => {
+const ErrorMessage = ({ error, message }) => {
```

Comment:
@binh-dam-ibigroup I noticed mapping the error message in mapState with injectIntl wouldn't work bc the ownProps.intl was not defined yet. Moved injectIntl to a useIntl hook and refactored.

Reply:
Tests will continue to fail pending some FormattedTime fixes I have to make in the main branch so if this change is ok, I'll merge this and keep working on the `react-intl-ramp-up-pr3` branch